mLRS/Common/bind.h

```diff
@@ -127,6 +129,42 @@ void tBindBase::ConfigForBind(void)
     sx2.SetToIdle();
 }
 
+void tBindBase::HopToNextBind(uint8_t frequency_band)
+{
+#ifdef DEVICE_HAS_LR11xx
```

Comment:
for the moment ok, but I think we want to have something more generic/general
we want to try to have as little sx-dependencies as possible
so, we could add a comment for now, `// TODO: replace by some more generic flag`